tests/tests.cpp

```diff
@@ -9,23 +10,85 @@
 
 extern "C" bool helloWorld;
 extern "C" bool func_1();
+
+/// Externals used for testing [Issue #7]: https://github.com/Team-10-But-Better/Craft/issues/7
+extern "C" double mouseDPIMultiplier;
+extern "C" double keyDPIMultiplier;
+extern "C" void incrementDPI();
+extern "C" void decrementDPI();
+extern "C" void setDefaultDPI();
+
+/// Externals used for testing [Issue #41]: https://github.com/Team-10-But-Better/Craft/issues/41
+extern "C" int isDefinedCraftKey(char *str, int start, int end);
+
 //----------------------------------------------------
-TEST(Assert, empty) { GTEST_ASSERT_EQ(true, true); }
+TEST(Assert, empty)
+{
+	GTEST_ASSERT_EQ(true, true);
+}
 
 //----------------------------------------------------
-TEST(tests, issueFoo){
+TEST(tests, issueFoo)
+{
 	EXPECT_EQ(helloWorld, true);
 }
-TEST(tests, issueBar){
+
+TEST(tests, issueBar)
+{
 	bool returnValue = func_1;
 	EXPECT_EQ(true, returnValue);
 }
-extern "C" int craft_main(int argc, char* argv[]);
+
+TEST(tests, issue7)
```

Comment:
This is Issue 59's Pull request, why is this test being added for issue 7? This seems wrong